app/views/layouts/application.html.erb

```diff
@@ -1,16 +1,16 @@
 <!doctype html>
 <html>
-<head>
-  <title>Buildlight by Collective Idea</title>
-  <%= stylesheet_link_tag    "application", :media => "all" %>
-  <%= javascript_include_tag "application" %>
-  <%= csrf_meta_tags %>
-</head>
-<body>
+  <head>
+    <meta charset="utf-8" />
+    <meta content="IE=edge" http-equiv="X-UA-Compatible" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>[i] Buildlight</title>
+    <link rel="shortcut icon" href="/favicon.ico" />
+    <%= stylesheet_link_tag "application", :media => "all" %>
+    <%= csrf_meta_tags %>
+  </head>
 
-<span id="forkongithub"><a href="https://github.com/collectiveidea/buildlight">Fork me on GitHub</a></span>
-
-<%= yield %>
-
-</body>
+  <body<% if defined? @colors %><%= " data-failing" if @colors[:red] %><%= " data-building" if @colors[:yellow] %><%= " data-passing" if @colors[:green] %><% end %>>
```

Comment:
These three data attributes are what the styles rely on to position/toggle things into/out of view. Just FYI.